### PR TITLE
Add debianization for Bullseye [REVPI-3011]

### PR DESCRIPTION
**Pull request by nbuchwitz**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -0,0 +1,31 @@
+revpi-wallpaper (1.0.0-5) buster; urgency=medium
```
Comment (naruxde) [inline debian/changelog]:
```suggestion
revpi-wallpaper (1.0.0-5+revpi11+1) buster; urgency=medium
```

**Review by naruxde — CHANGES_REQUESTED** [review]
I think this branch should start from the clean raspios/buster branch. Changes requested in PR for raspios/buster are the same in this PR

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Source: revpi-wallpaper
+Section: contrib/electronics
+Priority: optional
+Rules-Requires-Root: no
+Build-Depends: debhelper-compat (= 13)
+Standards-Version: 4.5.1
+Maintainer: KUNBUS GmbH <support@kunbus.com>
+Homepage: https://revolution.kunbus.com/
```
Comment (naruxde) [inline debian/control]:
```suggestion
Homepage: https://revolutionpi.com/
```

**Review by naruxde — CHANGES_REQUESTED** [review]

**Comment by nbuchwitz** [comment]
Rebase to raspios/buster after merge of #1

**Pull request merged**